src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -7013,11 +7002,7 @@ record C1
 ";
 
             var comp = CreateCompilation(src);
-            comp.VerifyEmitDiagnostics(
-                // (4,35): error CS8870: 'C1.ToString()' cannot be sealed because containing record is not sealed.
-                //     public sealed override string ToString() => throw null;
-                Diagnostic(ErrorCode.ERR_SealedAPIInRecord, "ToString").WithArguments("C1.ToString()").WithLocation(4, 35)
```

Comment:
This is no longer illegal, which is the goal of this PR.

Reply:
The new feature should be gated on a language version, so a diagnostic should remain for C# 9 in this scenario. The diagnostic will change though (it should say "to use this feature, you should use language version "preview"").

This means adding a new feature flag. Here's an existing example: `MessageID.IDS_FeatureLambdaDiscardParameters.CheckFeatureAvailability(...)`

Reply:
Ah, yes. Sorry, forgot about that (it's my first contribution on the compiler). I'll look into it.

Reply:
Done